UM/View/GL/OpenGL.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2015 Ultimaker B.V.
+# Cura is released under the terms of the AGPLv3 or higher.
+
+##  Convenience methods for dealing with OpenGL.
+#
+#   This class simplifies dealing with OpenGL and different implementations
+#   of it. It mostly describes an interface that should be implemented for
+#   different implementations. Additionally, it provides singleton handling.
```

Comment:
Weird wording. Implemented for different implementations. Might want to rewrite that a bit.